ansible/roles_ocp_workloads/ocp4_workload_cyberark_dap/tasks/workload.yml

```diff
@@ -0,0 +1,134 @@
+---
+- name: Setting up CyberArk DAP workload for user
+  debug:
+    msg: "Setting up CyberArk DAP workload for user ocp_username = {{ ocp_username }}"
+
+# CyberArk DAP workload tasks
+# -------------------------------------------------------------------------
+
+- name: Create the cyberark setup directory
+  file:
+    path: /home/{{ansible_user}}/cyberark_setup
+    state: directory
+
+- name: Download the CyberArk RHPDS setup files
+  become: true
+  unarchive:
+    src: "{{ ocp4_workload_cyberark_setup_files_url }}"
+    dest: /home/{{ansible_user}}/cyberark_setup
+    remote_src: True
+    owner: "{{ ansible_user }}"
+    group: users
+
+- name: Set random password for dap
+  set_fact:
+    dap_pwd: "{{ lookup('password', '/dev/null length=41 chars=ascii_letters,digits,hexdigits,@,#,$,%,^,*,') }}"
+
+- name: create CyberArk resources
+  k8s:
+    state: present
+    definition: "{{ lookup('template', item ) | from_yaml }}"
+  loop:
+    - ./templates/namespace.j2
+    - ./templates/service.j2
+    - ./templates/dap_authn_role.j2
+    - ./templates/role_binding.j2
+    - ./templates/dap_cm_role.j2
+    - ./templates/secrets_cluster_role.j2
+
+- name: Add policy add-scc-to-user dap authn service
+  command: "oc adm policy add-scc-to-user anyuid -z dap-authn-service -n {{ ocp4_workload_cyberark_project }}"
```

Comment:
Not required, but generally better to use Roles and RoleBindings to map SCCs to service accounts rather than directly manipulating the SCC definitions.

Reply:
Will look it to this, thanks.

Reply:
Done, I created an SCC template